src/dafny/DafnyCore.md

```diff
@@ -23,7 +23,7 @@ in the github repository [here](https://github.com/dafny-lang/libraries/tree/mas
 ## Content of the Core Library
 
 The sections below describe how to use each of the principal modules of the library:
-- [Dafny.Wrappers](Results) -- containers that enable encapsulaton of success and failure results in a single datatype
+- [Dafny.Wrappers](Wrappers) -- containers that encapsulate success and failure results in a single datatype
```

Comment:
This description doesn't match the content, I'm taking a stab at a different one...

Reply:
```suggestion
- [Dafny.Wrappers](Wrappers) -- simple datatypes to support common patterns, such as optional values or the result of operations that can fail.
```